Shared/mods/deathmatch/logic/luadefs/CLuaXMLDefs.cpp

```diff
@@ -184,6 +185,11 @@ int CLuaXMLDefs::xmlLoadFile ( lua_State* luaVM )
                             }
                         }
 
+                        xmlFile->GetLastError ( strError );
+                        const int exists = strstr ( strError, "Failed to open file" ) != NULL;
+                        if( exists==0 )
+                            argStream.SetCustomError ( strError, SString("Unable to read XML file %", strFileInput.c_str() ) );
```

Comment:
You are missing the format specifier **s** in the format string: `"Unable to read XML file %"` should be `"Unable to read XML file %s"`